code/modules/atmospherics/Atmospherics.md

```diff
@@ -157,14 +157,44 @@ As for react(), it is where all the behavior of the reaction is defined. The pro
 
 This is a rather large subject, we will need to cover gas flow, turf sleeping, superconduction, and much more. Strap in and enjoy the ride!
 
+### A Word On `Share()`
```

Comment:
shouldn't this be as long and as detailed as your PR comment? github won't be around forever.

Reply:
I thought it was honestly. They're both edited versions of one of my commits. Though I touched up the pr desc after I wrote this bit, so it may be missing some stuff. I'll do a second pass

Reply:
I had a go at improving it, lemme know if it needs some more touchups yeah?